packages/manager/src/features/CloudView/Dashboard/DashboardLanding.tsx

```diff
@@ -119,14 +118,14 @@ export const DashBoardLanding = () => {
       preferenceRef.current.aclpPreference.resources = globalFilter.resource;
     }
 
-    if (changedFilter === 'timestep') {
+    if (changedFilter === INTERVAL) {
       dashboardPropRef.current.dashboardFilters.interval =
         globalFilter.interval;
       dashboardPropRef.current.dashboardFilters.step = globalFilter.step;
       preferenceRef.current.aclpPreference.interval = globalFilter.interval;
```

Comment:
why we need this preferenceRef?, we need to remove this from dashboardlanding

Reply:
This will be / should be remove from here, so lot of boilerplate code is not needed. We will consider this later